stories/Templates/PublicationPage/publicationpage.scss

```diff
@@ -64,6 +78,10 @@
     margin-bottom: $spacing-24;
   }
 
+  &-36 {
```

Comment:
@neelesh-tippe this class is written twice, once on line #48 and line #81. Same goes for `&-40`. Can you write it once?

Reply:
Those are different classes for margin-top and margin-bottom with same space.